lib/rspec/legacy_formatters.rb

```diff
@@ -2,6 +2,7 @@
 require "rspec/legacy_formatters/version"
 require "rspec/legacy_formatters/adaptor"
 require 'stringio'
+require 'rspec/core'
```

Comment:
It's required on line 1... it needs to be there so that it's loaded *before* this, but this will have no effect...